scripts/compare.py

```diff
@@ -0,0 +1,27 @@
+import json
```

Comment:
@victorggonzalez, have you checked this file with PEP-8 style? I think there are some style issues (e.g., missing spaces before a }).

Reply:
Done! Used pycodestyle to perform the check and formatting